src/index.html

```diff
@@ -3,11 +3,281 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>Bose</title>
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="images/favicon.png"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
+
   <body>
-    <h1>Miami</h1>
+
+    <header class="header" id="home">
+      <nav class="navigation header__navigation">
+        <div class="logo__container">
+          <a class="logo navigation__item" href="#">
+            <img
+              class="logo__image"
+              src="/images/logo-bose.svg"
+              alt="logo bose">
+          </a>
+        </div>
+
+        <div class="icons__container">
+          <a class="phone navigation__item" href="tel:+1 234 555-55-55">
```

Comment:
Phone link o hover should display the phone number as in the designs:
<img width="307" alt="image" src="https://github.com/mate-academy/layout_miami/assets/126275137/f366563b-0ce0-450a-b2aa-7da280033304">